src/contractor.jl

```diff
@@ -40,9 +47,10 @@ function Base.show(io::IO, C::Contractor{N,Nout,F1,F2,ex}) where {N,Nout,F1,F2,e
 end
 
     (C::Contractor)(X) = C.forward(X)[1]
+    (C::BasicContractor)(X) = C.forward(X)[1]
 
-function (C::Contractor{N,Nout,F1,F2,ex})(
-    A::IntervalBox{Nout,T}, X::IntervalBox{N,T}) where {N,Nout,F1,F2,ex,T}
+function (C::Contractor{N,1,F1,F2,ex})(
```

Comment:
You can remove all the type parameters in this function, since we don't need them